From a50e322b8104327e5538f6168d4c5a4dd5d230df Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ulrich=20M=C3=BCller?= <ulm@gentoo.org>
Date: Sun, 20 Dec 2015 20:21:48 +0100
Subject: Fix jpeg version check to work with gcc >= 5

Comprises part of the following commit from the emacs-24 branch:

commit ff3878d749591ebf78da532ec1eb6fa00cb5757d
Author: Andreas Schwab <schwab@suse.de>
Date:   Mon Mar 23 11:55:24 2015 +0100

    * configure.ac: Fix jpeg version check to work with gcc >= 5.

Origin: https://gitweb.gentoo.org/proj/emacs-patches.git/commit/?id=a522741e3d974f314fa96b3672739a2dbf0ca43c
---
 configure.in | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/configure.in b/configure.in
index 75af0f71ecc..84fc0dc0445 100644
--- a/configure.in
+++ b/configure.in
@@ -2140,9 +2140,11 @@ if test "${HAVE_X11}" = "yes"; then
   AH_TEMPLATE(HAVE_JPEG, [Define to 1 if you have the jpeg library (-ljpeg).])dnl
   if test "${HAVE_JPEG}" = "yes"; then
     AC_DEFINE(HAVE_JPEG)
-    AC_EGREP_CPP([version= *(6[2-9]|[7-9][0-9])],
+    AC_EGREP_CPP([version 6b or later],
         [#include <jpeglib.h>
-	 version=JPEG_LIB_VERSION
+	 #if JPEG_LIB_VERSION >= 62
+	 version 6b or later
+	 #endif
 ],
         [AC_DEFINE(HAVE_JPEG)],
         [AC_MSG_WARN([libjpeg found, but not version 6b or later])
